WooCommerce/Classes/ViewRelated/Dashboard/StoreStats/StorePerformanceView.swift

```diff
@@ -0,0 +1,117 @@
+
+                timeRangeBar
+                    .padding(.horizontal, Layout.padding)
+                    .redacted(reason: viewModel.syncingData ? [.placeholder] : [])
+                    .shimmering(active: viewModel.syncingData)
+
+                Divider()
+
+            }
+            .padding(.vertical, Layout.padding)
+            .background(Color(.listForeground(modal: false)))
+            .clipShape(RoundedRectangle(cornerSize: Layout.cornerSize))
+            .padding(.horizontal, Layout.padding)
+            .sheet(isPresented: $showingCustomRangePicker) {
+                RangedDatePicker(startDate: viewModel.startDateForCustomRange,
+                                 endDate: viewModel.endDateForCustomRange,
+                                 datesFormatter: DatesFormatter(),
+                                 customApplyButtonTitle: viewModel.buttonTitleForCustomRange,
+                                 datesSelected: { start, end in
+                    viewModel.didSelectTimeRange(.custom(from: start, to: end))
+                })
+            }
+        } else {
+            ViewControllerContainer(DeprecatedDashboardStatsViewController())
+        }
+    }
+}
+
+private extension StorePerformanceView {
+    var header: some View {
+        HStack(alignment: .top) {
+            Text(Localization.title)
+                .headlineStyle()
+            Spacer()
+            Menu {
+                Button(Localization.hideCard) {
+                    // TODO
+                }
+            } label: {
+                Image(systemName: "ellipsis")
+                    .foregroundStyle(Color.secondary)
+            }
+        }
+    }
+
+    var timeRangeBar: some View {
+        HStack(alignment: .top) {
+            AdaptiveStack {
+                Text(viewModel.timeRange.tabTitle)
+                    .foregroundStyle(Color(.text))
+                    .subheadlineStyle()
+                Text(viewModel.timeRangeText)
+                    .subheadlineStyle()
+            }
+            Spacer()
+            StatsTimeRangePicker(currentTimeRange: viewModel.timeRange) { newTimeRange in
+                if newTimeRange.isCustomTimeRange {
+                    showingCustomRangePicker = true
+                } else {
+                    viewModel.didSelectTimeRange(newTimeRange)
+                }
+            }
+        }
+    }
+}
+
+private extension StorePerformanceView {
+    enum Layout {
+        static let padding: CGFloat = 16
+        static let cornerSize = CGSize(width: 8.0, height: 8.0)
+        static let strokeWidth: CGFloat = 0.5
+    }
+
+    enum Localization {
+        static let title = NSLocalizedString(
+            "storePerformanceView.title",
+            value: "Performance",
+            comment: "Title of the store performance section on the Dashboard screen"
+        )
+        static let hideCard = NSLocalizedString(
+            "storePerformanceView.hideCard",
+            value: "Hide this card",
```

Comment:
Discussion: I wonder if this should say "Hide Performance" (or "Hide Card" if we want to make it generic for other cards too). This is because for Blaze, the text is "Hide Blaze":

<img width="219" alt="Screenshot 2024-04-16 at 20 08 09" src="https://github.com/woocommerce/woocommerce-ios/assets/266376/7151103f-711a-4853-8189-89405a1c8c15">

Reply:
Updated the wording in 7b8b2af6b0fe071043d97c71f6ccfdaf86f43ff2.